server/api.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost/server/public/plugin"
+)
+
+// ServeHTTP demonstrates a plugin that handles HTTP requests by greeting the world.
```

Comment:
Propose we document the fully qualified path to hit this endpoint -- this came up in a recent hackathon with a prospect who asked how to hit the API.